lib/fs.js

```diff
@@ -1305,11 +1311,17 @@ function FSWatcher() {
   this._handle.owner = this;
 
   this._handle.onchange = function(status, eventType, filename) {
+    // TODO(joyeecheung): we may check self._handle.initialized here
```

Comment:
cc @bnoordhuis can you check that this can be fixed this way? Thanks!

Reply:
BTW, this is coming from:
https://github.com/nodejs/node/blob/9e9c516512668133813c71e39d56b011b804572b/src/fs_event_wrap.cc#L161-L171